lib/yrewrite/seo.php

```diff
@@ -61,21 +61,66 @@ public function __construct($article_id = 0, $clang = null)
         }
     }
 
+    public function getTags()
+    {
+        $tags = [];
+        $tagsOg = [];
+        $tagsTwitter = [];
+        $tagsTwitter['twitter:card'] = '<meta name="twitter:card" content="summary" />';
+
+        $title = rex_escape(strip_tags($this->getTitle()));
```

Comment:
War vorher schon so, aber `strip_tags` finde ich hier verkehrt. Vielleicht in der neuen Methode hier drauf verzichten.
Wenn der Titel `Erklärung des <picture>-Elements` heißt, dann soll der Tag nicht entfernt werden, sondern nur escaped werden.

Gilt auch für die Description.